src/Sylius/Bundle/ResourceBundle/Controller/RequestConfiguration.php

```diff
@@ -92,6 +92,23 @@ public function isHtmlRequest()
     }
 
     /**
+     * @return integer
+     */
+    public function getResponseCode()
+    {
+        $responseCode = $this->parameters->get('response_code');
+
+        if (is_array($responseCode)) {
+            $controller = $this->request->attributes->get('_controller');
+            if (preg_match('/^.+\:(.+)Action$/i', $controller, $matches)) {
+                return $responseCode[$matches['1']];
```

Comment:
I will improve this part and also want to cover different cases in phpspec. Not even sure if I want to even use a reg ex.